DirectProgramming/C++SYCL_FPGA/Tutorials/Features/loop_coalesce/README.md

```diff
@@ -1,14 +1,27 @@
 
-# Coalescing Nested Loops
-This FPGA tutorial demonstrates applying the `loop_coalesce` attribute to a nested loop in a task kernel to reduce the area overhead.
+# `Loop Coalesce` Sample
```

Comment:
```suggestion
# `loop_coalesce` Sample
```
?

Reply:
I feel like we should either do:
1/ Loop Coalesce Sample
or
2/ `loop_coalesce` Sample